cc/algorithms/count_test.cc

```diff
@@ -192,5 +194,29 @@ TEST(CountTest, MemoryUsed) {
   EXPECT_GT((*count)->MemoryUsed(), 0);
 }
 
+TEST(CountTest, DeltaNotSetGaussian) {
+  auto failed_count = Count<double >::Builder()
+          .SetEpsilon(0.5)
+          .SetLaplaceMechanism(absl::make_unique<differential_privacy::GaussianMechanism::Builder>())
+          .Build();
+  EXPECT_THAT(failed_count,
+  StatusIs(absl::StatusCode::kInvalidArgument,
+  HasSubstr("Delta has to be set")));
+}
+
+TEST(CountTest, BasicGaussian) {
+  std::vector<int> c = {1, 2, 3, 4, 2, 3};
+  auto count =
+      typename Count<int>::Builder()
+          .SetEpsilon(10)
+          .SetDelta(1e-5)
+          .SetLaplaceMechanism(absl::make_unique<differential_privacy::GaussianMechanism::Builder>())
```

Comment:
Just double-checking something. "SetLaplaceMechanism" is the deprecated method IIRC.
There is no "SetMechanism" method yet, right? If it has already been added, then I believe we should use it here and in the other test.

Reply:
I already have a branch for that, But Chris and I decided to push it later.

Reply:
We can deprecate it later when we found a good solution for choosing the noise in the builder.